spec/chromium-spec.js

```diff
@@ -1411,6 +1411,83 @@ describe('chromium feature', () => {
       await new Promise((resolve) => { utter.onend = resolve })
     })
   })
+
+  describe('focus handling', () => {
```

Comment:
Is chromium-spec.js the best place for this test?

Reply:
Yup it looks good since we aren't testing any webContent specific apis.